src/styles/components/_override.buttons.scss

```diff
@@ -29,6 +29,7 @@
 
   &.ds-c-button--transparent {
     border-color: transparent;
+    background-color: transparent;
```

Comment:
I'm wondering about this change - what happens if this transparent button is used on a dark background? We had issues in the past with using true `transparent` buttons which is why white was used for the background because most use cases were on a white background.

Reply:
This was an issue I saw show up when getting the testing environment running on for the coverage-tools-frontend where MCT is using the transparent button with alerts and other background colors. In PROD they don't use a white background for transparent buttons. If we added the white background, the screenshots below is what their transparent buttons would look like in PROD. 

![image](https://user-images.githubusercontent.com/4505990/128770083-912b0d8e-3094-45f8-955c-cecffce10f03.png)
![image](https://user-images.githubusercontent.com/4505990/128770136-30e50819-6092-4441-91d7-cc048c7ce9fa.png)

Reply:
Thanks for the context, that makes sense!